libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -1265,6 +1268,204 @@ private static string AddOrdinal(int num)
             return ordinalResult;
         }
 
+        // conversion functions
+        private static string ToBinary(string strToConvert)
+        {
+            var result = string.Empty;
+            foreach (var element in strToConvert.ToCharArray())
+            {
+                result += Convert.ToString(element, 2).PadLeft(8, '0');
+            }
+
+            return result;
+        }
+
+        private static (object, string) ToXml(object contentToConvert)
+        {
+            object result = null;
+            string error = null;
+            try
+            {
+                if (contentToConvert is string str)
+                {
+                    var xml = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(Encoding.ASCII.GetBytes(str), new XmlDictionaryReaderQuotas()));
+                    result = xml.ToString().TrimStart('{').TrimEnd('}');
+                }
+                else
+                {
+                    var xml = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(Encoding.ASCII.GetBytes(contentToConvert.ToString()), new XmlDictionaryReaderQuotas()));
+                    result = xml.ToString().TrimStart('{').TrimEnd('}');
```

Comment:
`result = xml.ToString().TrimStart('{').TrimEnd('}');` this function can be extracted from if() else() condition